static/src/js/components/Header/Header.jsx

```diff
@@ -69,7 +69,7 @@ const Header = ({ noLogin }) => {
                     onClick={login}
                     variant="blue-light"
                   >
-                    Log in with Launchpad
+                    Log in with EDL
```

Comment:
Log in to MMT

Reply:
I never changed this, as I couldn't remember if a decision was made on what to call this.